diff --git a/showfont.c b/showfont.c
index 4ba01a6..452b668 100644
--- a/showfont.c
+++ b/showfont.c
@@ -356,7 +356,7 @@ usage(void)
 int
 main(int argc, char **argv)
 {
-    const char *servername = "localhost:7100"; /* -server: font server name */
+    const char *servername = "unix/:-1"; /* -server: font server name */
     char *fontname = NULL; /* -fn: font name */
     int         i;
     Font        fid,
